src/js/themes/base.js

```diff
@@ -278,6 +278,32 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       },
       // extend: undefined,
     },
+    audio: {
+      icons: {
+        pause: Pause,
+        play: Play,
+        volume: Volume,
+        color: 'white',
+      },
+      volume: {
+        background: 'light-4',
+        color: 'brand',
+        opacity: 'strong',
+      },
+      controls: {
+        volume: {
```

Comment:
Can we have all of the volume stuff in `audio.volume`. It seems strange to have some volume stuff under `controls`.

Reply:
of course. done.